### PR TITLE
Bug 2079 - Fix share URL in post roll

### PR DESCRIPTION
**Pull request by jbuck**

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/1f2ca7a32214ef2/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/1f2ca7a32214ef2/output.txt

Total script time: 0.94 mins

Server is running on http://pr859.butyr.org/templates/basic/
Run the Test Suite: http://pr859.butyr.org/test/

File: cornfield/views/embed.jade [hunk 1]
```diff
@@ -55,7 +55,7 @@ html(lang='en')
             #share-options.share-options
               fieldset
                 label URL
-                input#share-url.share-ul(type='url', value='http://wwww.gasdasdasdasd.com')
```
Comment (cadecairos) [inline cornfield/views/embed.jade]: lol wut.

**Pull request merged**